Sources/Layout.swift

```diff
@@ -166,18 +163,17 @@ extension View {
    - returns: The constraint that was added
    */
   @discardableResult
-  public func size(axis: Axis, relatedBy relation: LayoutRelation, size: CGFloat, priority: LayoutPriority = LayoutPriorityRequired) -> NSLayoutConstraint {
-    var constraint = Constraint(view: self)
-    
-    constraint.firstAttribute = sizeAttribute(axis: axis)
-    constraint.secondAttribute = sizeAttribute(axis: axis)
-    constraint.relation = relation
-    constraint.constant = size
+  public func size(axis: Axis, relatedBy relation: LayoutRelation, size: CGFloat, priority: LayoutPriority = .required) -> NSLayoutConstraint {
+    let constraint = NSLayoutConstraint(item: self,
+                                        attribute: sizeAttribute(axis: axis),
+                                        relatedBy: relation,
+                                        toItem: nil,
+                                        attribute: .notAnAttribute,
```

Comment:
I've changed the second attribute to `.notAnAttribute` from `sizeAttribute(axis: axis)` because if the `toItem` is nil then it's set to `.notAnAttribute` no matter the value you pass in.